integration_test/cases/lock.exs

```diff
@@ -15,17 +15,17 @@ defmodule Ecto.Integration.LockTest do
   end
 
   setup do
-    %LockCounter{id: 42, count: 1} |> PoolRepo.insert
+    %LockCounter{count: 1} |> PoolRepo.insert
 
     on_exit fn ->
-      PoolRepo.get(LockCounter, 42) |> PoolRepo.delete
+      PoolRepo.get(LockCounter, 1) |> PoolRepo.delete
```

Comment:
I don't think we can rely on the ID being one. So instead we should do somthing like:

```elixir
lc = PoolRepo.insert %LockCounter{count: 1}

on_exit fn -> PoolRepo.delete(lc) end

{:ok, lc: lc}
```

Thoughts?